README.md

```diff
@@ -100,51 +90,6 @@ Every component on A-Frame has a series of methods to be extended to determine i
 * Init: Sets the initial state of the entity. Also, It loads the shaders and transfer functions.
 * Tick: check for events in the vr controllers to produce real time interaction with the data.
 
-
-#### Structure of the project
-
-The key folders of the project are ‘public’ and ‘src’
-
-![file_system1](./imgs/filesystem.png)
-
-The ‘public’ folder contains the index.html file which is initially loaded when we the user enters to the web application.
-
-![file_system1](./imgs/filesystem2.png)
-
-The A-Frame components are located in the ‘src’ folder:
-
-![file_system1](./imgs/filesystem3.png)
-
-#### How to access it from the ccv server
-
-Open your web browser and go to https://datasci.brown.edu/spheroid/ or https://datasci.brown.edu/simulation/.
-
-The VR mode is only supported by Mozilla FireFox. Please install this application if you want to use HMD device to visualize the data.
-
 #### How the application works
 
```

Comment:
This can be deleted too since the section is gone

```suggestion
```

Reply:
I added a link to the instructions section in the web page.